react/src/pages/UserEditPage.js

```diff
@@ -74,8 +77,8 @@ const UserEditPage = () => {
     return <>
         <center>
             <div style={{display:"table-row"}}>
-                <b style={{paddingLeft:"0.2em",float:"left"}}>Edit User</b>
```

Comment:
Is there a reason you're specifying this padding in ems when for so many other things you use pts or pxs?

Reply:
This particular one is removed now with a bit of simplification. My understanding is that em spacing is somewhat preferred for padding, but again I've much to learn here stil.